File: python/pad_array.py
def pad(array, min_size, value = None):
    x = min_size - len(array)

    while x > 0:
        array.append(value)
        x -= 1
    return array


# print(pad([1,2,3], 5) == [1,2,3,None,None])

File: js/padArray.js
exports.padArray = function (arr, minSize, value=undefined) {

    if (arr.length < minSize) {
        let answer = arr
        for (let i = 0; minSize > arr.length; i++) {
            answer.push(value)
        }
        return answer
    } else {
        return arr
    }  

}
console.log(this.padArray([1,2,3],5))
